JavaIntegration/src/main/java/org/openzen/zencode/java/impl/conversion/JavaNativeHeaderConverter.java

```diff
@@ -83,21 +83,15 @@ public FunctionHeader getHeader(
 				typeParameters[i].addBound(new ParameterTypeBound(CodePosition.NATIVE, typeConverter.getType(context, bound)));
 		}
 
-		FunctionParameter[] parameters = new FunctionParameter[javaParameters.length];
-		int classParameters = 0;
-		for (int i = 0; i < parameters.length; i++) {
-			Parameter parameter = javaParameters[i];
-			if (parameter.getType().getCanonicalName().contentEquals("java.lang.Class")) {
-				classParameters++;
-			}
-
-			TypeID type = typeConverter.getType(context, parameter.getAnnotatedType());
-			parameters[i] = new FunctionParameter(type, parameter.getName(), parameter.isVarArgs());
-			parameters[i].defaultValue = getDefaultValue(parameter, type, parameters[i]);
-		}
-		if (classParameters > 0 && classParameters == typeParameters.length) {
-			parameters = Arrays.copyOfRange(parameters, classParameters, parameters.length);
-		}
+		FunctionParameter[] parameters = Arrays.stream(javaParameters)
```

Comment:
Not a nice impl, but for now it makes all tests pass (TDD for the win 😉 )  
Just skips all Class parameters, regardless of where in the signature.